ts-apps/jl4-web/src/routes/+layout.svelte

```diff
@@ -0,0 +1,6 @@
+<script>
+  import '../style.css';
+  import '../../../decision-logic-visualizer/src/style.css'
```

Comment:
It's not a good idea to import the CSS from the dlv package directly like this. It's better to do it in the style of

https://github.com/smucclaw/l4-ide/blob/cb9aa6a430d3357b46cd4962416508b6711ea858/ts-apps/webview/src/app.css#L5

where we 

(i) import it from [the export path that's declared in the dlv package.json](https://github.com/smucclaw/l4-ide/blob/cb9aa6a430d3357b46cd4962416508b6711ea858/ts-apps/decision-logic-visualizer/package.json#L29) and 

(ii) use the path alias supplied by turborepo.

(It probably doesn't matter whether you import it in the script or as an `@import` in the app.css, but deviating from (i) and (ii) may lead to unexpected or undefined behavior.)

Reply:
I'm happy to try changing it -- my personal inclination would be to remove Tailwind (as noted in https://github.com/smucclaw/l4-ide/pull/161#discussion_r1972933227) unless you have an independent reason for wanting it

Reply:
yes - less is better, if you manage to remove it that would be awesome!

Reply:
I'll wrok on it!

Reply:
(I'll also look into the export of the css by the `decision-logic-visualizer` package --- I might need to do more to get the styles correctly exported.)